chromium_src/components/permissions/permission_context_base.cc

```diff
@@ -19,7 +19,8 @@
 namespace {
 
 bool IsGroupedPermissionType(ContentSettingsType type) {
-  return type == ContentSettingsType::BRAVE_ETHEREUM;
+  return type == ContentSettingsType::BRAVE_ETHEREUM ||
+         type == ContentSettingsType::BRAVE_SOLANA;
```

Comment:
I'm a bit concerned about a proliferation of content setting types for this kind of stuff. Is there anything we can do in the url instead?